pallets/handles/src/handles-utils/src/suffix.rs

```diff
@@ -33,15 +33,13 @@ pub fn generate_unique_suffixes(
 	canonical_base: &str,
 ) -> impl Iterator<Item = u16> + '_ {
 	let seed = generate_seed(canonical_base);
-	let mut rng = SmallRng::seed_from_u64(seed);
```

Comment:
Maybe I am missing it, but I do not see any test for this function.

Reply:
It is forthcoming.  There were no suffix tests but now there are.